docs/tutorial/package.md

```diff
@@ -44,41 +44,36 @@ cd ./rick-portal-gun
 
 ## Dependencies and environment
 
-Add `typer[all]` to your dependencies:
+Add `typer` to your dependencies:
 
 <div class="termy">
 
 ```console
-$ poetry add "typer[all]"
+$ poetry add typer
 
 // It creates a virtual environment for your project
 Creating virtualenv rick-portal-gun-w31dJa0b-py3.10 in /home/rick/.cache/pypoetry/virtualenvs
-Using version ^0.1.0 for typer
+Using version ^0.12.0 for typer
 
 Updating dependencies
 Resolving dependencies... (1.2s)
 
-Writing lock file
-
 ---> 100%
 
-Package operations: 15 installs, 0 updates, 0 removals
-
-  - Installing zipp (3.1.0)
-  - Installing importlib-metadata (1.5.0)
-  - Installing pyparsing (2.4.6)
-  - Installing six (1.14.0)
-  - Installing attrs (19.3.0)
-  - Installing click (7.1.1)
-  - Installing colorama (0.4.3)
-  - Installing more-itertools (8.2.0)
-  - Installing packaging (20.3)
-  - Installing pluggy (0.13.1)
-  - Installing py (1.8.1)
-  - Installing shellingham (1.3.2)
-  - Installing wcwidth (0.1.8)
-  - Installing pytest (5.4.1)
-  - Installing typer (0.0.11)
+Package operations: 10 installs, 0 updates, 0 removals
+
+  - Installing mdurl (0.1.2)
+  - Installing markdown-it-py (3.0.0)
+  - Installing pygments (2.17.2)
+  - Installing click (8.1.7)
+  - Installing rich (13.7.1)
+  - Installing shellingham (1.5.4)
+  - Installing typing-extensions (4.10.0)
+  - Installing typer-slim (0.12.0)
+  - Installing typer-cli (0.12.0)
+  - Installing typer (0.12.0)
```

Comment:
Thanks for checking the changes and for the feedback @svlandeg! I have checked the installation of Typer 0.12.3 on my end and what I see is:
```
Package operations: 8 installs, 0 updates, 0 removals

  - Installing mdurl (0.1.2)
  - Installing markdown-it-py (3.0.0)
  - Installing pygments (2.17.2)
  - Installing click (8.1.7)
  - Installing rich (13.7.1)
  - Installing shellingham (1.5.4)
  - Installing typing-extensions (4.11.0)
  - Installing typer (0.12.3)
```
So `typer-slim` and `typer-cli` are removed on the way from 0.12.0 to 0.12.3, but `shellingham` is still present after running `poetry add typer` on Python 3.10, 3.11, and 3.12, making 8 installs instead of 7 noted on your end.

Reply:
Good to know! I was suprised that `shellingham` wasn't mentioned on my system, I'll have a look why that is. For this PR, I suggest you remove the references to `typer-slim` and `typer-cli`, as this is the current Typer installation behaviour that should be stable in the near future 🙂

Reply:
Right, `typer-slim` and `typer-cli` libraries are removed.